.ci/no-exception-test.sh

```diff
@@ -53,6 +53,28 @@ guava-with-sun-checks)
   rm sun_checks.*
   ;;
 
+openjdk14-with-checks-nonjavadoc-error)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: $CS_POM_VERSION
+  mkdir -p .ci-temp/
+  cd .ci-temp/
+  git clone https://github.com/checkstyle/contribution
+  cd ..
+  mvn -e clean install -Pno-validations
+  sed -i.'' 's/value=\"error\"/value=\"ignore\"/' \
+        .ci-temp/contribution/checkstyle-tester/checks-nonjavadoc-error.xml
+  cd .ci-temp/contribution/checkstyle-tester
+  sed -i '/  <!-- Filters -->/r ../../../.ci/openjdk14-excluded.files' checks-nonjavadoc-error.xml
```

Comment:
awesome hack ! I like it.
I would keep it in mind, it might be handy for other complicated cases also.

Reply:
I'll leave this here for the future then...
Ran from Checkstyle root:

```
#!/bin/bash
OUTPUT="test"
while  [ -n "$OUTPUT" ]
do
    OUTPUT=$(.ci/no-exception-test.sh openjdk14-checkstyle-checks-no-exception | grep "Caused by: " | grep -oh '/.*.java' | sed 's|.*/||' | cut -f 1 -d '.' | uniq)
    echo "$OUTPUT"
    echo -e "<module name=\"BeforeExecutionExclusionFileFilter\">\n  <property name=\"fileNamePattern\" value=\"/$OUTPUT\.java\$\"/>\n</module>" >> config/openjdk14-excluded.files
    rm -rf .ci-temp/contribution
done
```

Reply:
I've linked to the above script from the generated file in this PR, in case we should need to create such a file in the future.

Reply:
@nmancus1 , see my comment below on how to improve.